utils/near-performance-metrics/src/stats_enabled.rs

```diff
@@ -206,6 +239,10 @@ impl Stats {
             other_memory_size / MEBIBYTE,
             other_memory_count
         );
+        let rocksdb_memory_usage = get_rocksdb_memory_usage();
+        if rocksdb_memory_usage > 0 {
+            info!("    RocksDB memory usage: {}MiB", rocksdb_memory_usage / MEBIBYTE);
```

Comment:
You might want to steal the `Bytes` type from rust-analyzer to avoid `/ MEBIBYTE` on two different call-sites: 

https://github.com/rust-analyzer/rust-analyzer/blob/20a911f3cc2beb0409ab71cc1560648374745f7f/crates/profile/src/memory_usage.rs#L43-L80

In general, I've found that in Rust adding small well typed structs for quantities pays of. With derives, the boilerplate is relatively slim.

Reply:
How about we use https://github.com/hyunsik/bytesize/blob/master/src/lib.rs
Their code seems small and clean.

Reply:
Agree, that one is a solid crate!